src/components/DeleteConditionButton.vue

```diff
@@ -1,16 +1,18 @@
 <template>
 	<Button
+		iconOnly
 		class="delete-condition-button"
-		type="destructive"
-		variant="primary"
+		type="normal"
```

Comment:
This should be `type="neutral"` not normal, sorry!

Reply:
Yeah, noticed it late :D